--- a/cmake/compiler_flags_Intel_C.cmake
+++ b/cmake/compiler_flags_Intel_C.cmake
@@ -1,6 +1,4 @@
 # Intel C
-set( CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -sox -traceback")
-
 set( CMAKE_C_FLAGS_RELEASE "-qno-opt-dynamic-align -O2 -debug minimal -fp-model source")
 
 set( CMAKE_C_FLAGS_DEBUG "-O0 -g -ftrapuv")
--- a/cmake/compiler_flags_Intel_Fortran.cmake
+++ b/cmake/compiler_flags_Intel_Fortran.cmake
@@ -1,11 +1,8 @@
 # Precision-based Fortran compiler flags
 set(r4_flags "-real-size 32") # Fortran flags for 32BIT precision
 set(r8_flags "-real-size 64") # Fortran flags for 64BIT precision
-set(r8_flags "${r8_flags} -no-prec-div -no-prec-sqrt")
 
 # Intel Fortan
-set(CMAKE_Fortran_FLAGS "${CMAKE_Fortran_FLAGS} -fpp -fno-alias -auto -safe-cray-ptr -ftz -assume byterecl -align array64byte -nowarn -sox -traceback")
-
 set(CMAKE_Fortran_FLAGS_RELEASE "-O2 -debug minimal -fp-model source -nowarn -qoverride-limits -qno-opt-dynamic-align -qopt-prefetch=3")
 
 set(CMAKE_Fortran_FLAGS_DEBUG "-g -O0 -check -check noarg_temp_created -check nopointer -warn -warn noerrors -fpe0 -ftrapuv")
